### ISSUE TITLE
surface left behind after dismantling

### ISSUE THREAD

**Issue by MaddTrker** [MaFi-Games/Captain-of-Industry-issues#1445]
I had a power generation plant that I had started deconstructing, and after everything was hauled away or removed with unity part of the base structure was left behind and won't delete. I have tried duplicating the issue myself and it hasn't worked for me. 

![Screenshot (2)](https://github.com/MaFi-Games/Captain-of-Industry-issues/assets/166687303/cb492ceb-5d47-46c9-9a55-a9cc18dad42d)

**Comment by MaFi-Games** [comment]
Does this persist when you reload the save file? If yes, could you please share the file to us?

**Comment by MaddTrker** [comment]
yes, the surface is still there when opening the game several days later. I'm having issues getting the save file for the current playthrough. When I try pasting the file here it says you don't support my save file type. Please explain how to get the correct svae file. The file type I have is a .SAVE file. If there is a different save file that I should be looking for I'll need to know where to find that file type. Thank you for your time in this matter.

**Comment by elsender** [comment]
@MaddTrker 

To upload the save file,  you will need to Zip it first.
Right click on the file, Send to Compressed (zipped) file.   
it will make a copy of the file that is zipped.  
which you can then upload here.

**Comment by MaddTrker** [comment]
[New Haven 10.zip](https://github.com/MaFi-Games/Captain-of-Industry-issues/files/14980030/New.Haven.10.zip)

**Comment by MaddTrker** [comment]
I noticed there is another place on my map that the surface tiles have been left behind after dismatling a structure or object in the game. I have attached a ss of those surface tiles too. 

![Screenshot (2)](https://github.com/MaFi-Games/Captain-of-Industry-issues/assets/166687303/6dff3408-fd86-4983-9582-8a71a1ed025d)

**Comment by bknhrst** [comment]
i'm having the same thing happening after i removed supporting walls with surfaces between them. "remove surfaces" doesn't consider them valid and the delete tool doesn't work either. i'm running the most currently available update
[github.zip](https://github.com/user-attachments/files/15546017/github.zip)
